modules/understanding-dns-operator.adoc

```diff
@@ -0,0 +1,21 @@
+// Module filename: nw-creating-secured-routes.adoc
+// Module included in the following assemblies:
+// * networking/dns-operator.adoc
+
+[id="understanding-dns-operator-{context}"]
+= DNS Operator
+
+The DNS Operator deploys and manages CoreDNS to provide a name resolution
+service to pods that enables DNS-based Kubernetes Service discovery in
+OpenShift. It manages CoreDNS as a Kubernetes DaemonSet exposed as
+a Service with a static IP, and CoreDNS runs on all nodes in the cluster.
```

Comment:
This feels like a run-on sentence. 
_It manages CoreDNS as a Kubernetes DaemonSet exposed as a Service with a static **IP and runs CoreDNS  on** all nodes in the cluster._ ??

Reply:
For the one-liner, here's how [upstream describes it](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/):

>Kubernetes DNS schedules a DNS Pod and Service on the cluster, and configures the kubelets to tell individual containers to use the DNS Service’s IP to resolve DNS names.

Slightly modified for OpenShift might look like:

>OpenShift's DNS Operator manages a DNS Service on the cluster, and configures kubelets to tell individual containers to use the DNS Service’s IP to resolve DNS names.

An elaboration of this could include something about the DNS Service being deployed as a DaemonSet to ensure that each node hosts a DNS Service pod.

A further elaboration could include something about the DNS Service itself being implemented by CoreDNS using the Kubernetes plugin.